README.md

```diff
@@ -8,17 +8,25 @@ PokeReader is a 3gx plugin that allows viewing information about 3ds Pokemon gam
 
 This information can be used to RNG shiny and high IV Pokemon, similar to lua scripts on emulators.
 
+## Commands
+- X+Y to Lock/Unlock
+- D-pad keys to navigate when unlocked (right & left to get in and out of the different tabs)
+- Start + Select to pause game and enable manual frame advancing
+- Select to advance Main RNG frames once
+- Start or A to unpause
+- Start + Up (D-Pad) to Show/Hide the plugin
```

Comment:
Let's make this the first in the list since it feels like hiding the plugin is asked more than anything else.